notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>Instead of <code>EfdClient</code> , I recommend you use <code>makeEfdClient</code> . This will create an efd client independently if you are running this notebook at the summit or at USDF.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
OK!